<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="clientText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>No client</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clientBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pick a client</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="bookTab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section
{
   background-color: rgb(75, 75, 75);
	color: rgb(253, 253, 253);
}</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>102</number>
   </attribute>
   <column>
    <property name="text">
     <string>ISBN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Auther</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="dateText">
   <property name="geometry">
    <rect>
     <x>366</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>mm/dd/yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="invoiceNumberText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Invoice number: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(75, 75, 75);
color: rgb(253, 253, 253);</string>
   </property>
   <property name="text">
    <string>Confirm order</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
